core/node/via_btc_watch/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+use std::time::Duration;
+
+use anyhow::Context as _;
+use tokio::sync::watch;
+use via_btc_client::{indexer::BitcoinInscriptionIndexer, types::BitcoinTxid};
+use zksync_dal::{Connection, ConnectionPool, Core, CoreDal};
+
+use crate::message_processors::MessageProcessorError;
+
+mod message_processors;
+
+// re-exporting here isn't necessary, but it's a good practice to keep all the public types in one place
+pub use via_btc_client::types::BitcoinNetwork;
+
+#[derive(Debug)]
+struct BtcWatchState {
+    last_processed_bitcoin_block: u32,
```

Comment:
For now this is fine, but I believe when implementing the message processor we will need some kind of the serial number like `next_expected_message_id` to determine the next message to process.